azavea/raster-foundry
# Upgrade auth0 to auth.lock v10

## Overview

Upgrades auth0 to use angular-lock with auth0 lock v10. This adds support for extra signup fields.
I've included example fields, but we need to have a discussion on which fields we want / how we want to validate them.

### Demo

Coming once github calms down

### Notes

Lock v10 changes the styling classes, so I've made a few basic modifications and removed auth0 branding, but it doesn't quite match the theme of the application. @designmatty can clean it up when he has time.

## Testing Instructions

 * Verify that on page load, it requests a login
* Test the additional signup fields
* Verify that parts of the application which require authentication still work
* Verify that login state is preserved through page reloads
